docs/guides/server/configuration.adoc

```diff
@@ -101,6 +104,19 @@ You can also specify an explicit configuration file location using the `[-cf|--c
 
 <@kc.start rootParameters="--config-file=/path/to/myconfig.conf"/>
 
+=== Format for loading properties from a Java KeyStore
```

Comment:
Could you please also add the keystore in the example to be super explicit to users:
https://github.com/keycloak/keycloak/blob/7fccb6ab797c5af8028d0e22fc0243d970d23b8d/docs/guides/server/configuration.adoc?plain=1#L22

Reply:
thanks, suggestion applied